api/metrics/annually_test.py

```diff
@@ -0,0 +1,47 @@
+import sys
+sys.path.append("../")
+
+from api.metrics.annually import AnnuallyMetrics
+from api.utils.testing import create_test_db
+
+
+def test_annual_belonging_rate():
+    belonging_table = [
+        {
+            "layer": "place",
+            "period_end_year": 2015,
+            "segment": "all",
+            "value": 55.6
+        },
+        {
+            "layer": "place",
+            "period_end_year": 2016,
+            "segment": "all",
+            "value": 45.2
+        },
+        {
+            "layer": "place",
+            "period_end_year": 2017,
+            "segment": "all",
+            "value": 37.1
+        },
+        {
+            "layer": "place",
+            "period_end_year": 2018,
+            "segment": "all",
+            "value": 35.4
```

Comment:
I would suggest adding a row that does not have segment = "all" to make sure that it actually gets excluded.